drivers/sensor/lis2mdl/lis2mdl.c

```diff
@@ -307,13 +382,13 @@ static int lis2mdl_init(const struct device *dev)
 	}
 
 	if (wai != LIS2MDL_ID) {
-		LOG_DBG("Invalid chip ID: %02x\n", wai);
+		LOG_DBG("Invalid chip ID: %02x", wai);
```

Comment:
I think all of this changes are ok, but please do it in a separate commit first.
Like "fixing all LOG messages". And on top of that commit do this one that is introducing the SINGLE mode

Reply:
Well, actually are LOG_ERR(). But in any case do it in a preliminary fixing commit.

Reply:
I saw that you are changing all LOG_DBG in LOG_ERR in a separate commit, as requested.
But, are you really changing ALL the LOG_DBG in the driver? They seem to me only some of them.
Am I wrong? THis is the only part missing, than for me is ok.